### PR TITLE
Corrections to interval logic for PRMS restart mode run.

### PR DESCRIPTION
**Pull request by ashalper-usgs**

### CODE REVIEW

File: run [hunk 1]
```diff
@@ -194,14 +200,30 @@ for svc in out2ncf verifier; do
...
+
+# date (point) of the last simulation is the basename of the last
+# restart file
+p0=`ls $DIR/restart/*.restart | sed 's/^.*\///;s/\.restart$//' | \
       sort | tail -1`
 
-VAR_SAVE_FILE="-set var_save_file restart/$p.restart"
+p1=`date --date "$END_DATE" -59 days" --rfc-3339='date'`
+# if the restart point precedes ($END_DATE - 59 days) ...
+if [[ $p0 < $p1 ]];
```
Comment (rmcd-mscb) [inline run:213]: I think in this second run of PRMS the START_TIME will always be p0, but really I think it should be **p0 + 1 day**.

File: run [hunk 2]
```diff
@@ -194,14 +200,30 @@ for svc in out2ncf verifier; do
...
+then
+    # ... base case: start time is p0 (in PRMS start_time format)
+    START_TIME=`date --date $p0 +%Y,%m,%d,00,00,00`
+else
+    # ... model must have been run previously: start time is p1
+    START_TIME=$p1
+fi
+
+# end time is yesterday
+END_TIME=`date --date yesterday +%Y,%m,%d,00,00,00`
```
Comment (rmcd-mscb) [inline run:223]: END_TIME should be p0 + 1 day, it's just running a single day, and updating the restart file.

File: run [hunk 3]
```diff
@@ -194,14 +200,30 @@ for svc in out2ncf verifier; do
...
+
+# new restart file path, with basename ($END_DATE - 59 days)
+VAR_SAVE_FILE="-set var_save_file $DIR/restart/$p1.restart"
```
Comment (rmcd-mscb) [inline run:226]: again here the value of $p1 should be p0 + 1

Comment (rmcd-mscb) [reply run:226]: So in summary, in this second PRMS run, we increment the latest restart file date by 1 day, that is then the start_ and end_date.  PRMS runs for 1 day, using the most recently stored restart file to set the model state, the model is run for the following day, just one day, and writes a new restart file corresponding to the date of that file.

Comment (rmcd-mscb) [reply run:226]: Andy, I also think that in the code block 
``` bash
# if START_DATE is not set already
if [ "$START_DATE" = "" ]; then
    # if the most recent restart date is less than yesterday - 59 days...
    START_DATE=`date --date "$yesterday -59 days" +%Y-%m-%d`
    if [ $(date -d "$restart_date" +%s) -lt $(date -d "$START_DATE" +%s) ]; then
	# ...run PRMS from the most recent restart date
	START_DATE="$restart_date"
    fi
fi
```
START_DATE="$restart_date" should be the equivalent of "$restart_date" + 1 day

Sorry this is new but I just realized this is the way it should really be.

Comment (ashalper-usgs) [reply run:226]: Right. I just noticed that myself. It's redundant with the code I wrote today.

**Pull request closed without merging**